packages/browser-sdk/src/lib/core/redux/slices/rtcAnalytics.ts

```diff
@@ -207,7 +214,9 @@ startAppListening({
         const rtcManager = selectRtcConnectionRaw(state).rtcManager;
         if (!rtcManager) return;
 
-        const rtcCustomEvent = Object.values(rtcAnalyticsCustomEvents).find(({ action }) => action?.type === type);
+        const rtcCustomEvent = Object.values(rtcAnalyticsCustomEvents).find(({ actions }) =>
+            actions?.map((a) => a.type).includes(type)
```

Comment:
similar here 🙂

Reply:
Didn't change it here, as in my opinion it made it more unreadable in this case.